<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubDrive Configuration Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/settings.ico</normaloff>:/images/settings.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="connect_pushButton">
         <property name="text">
          <string>Connect to SubDrive</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/1415019116_Wireless_signal.png</normaloff>:/images/1415019116_Wireless_signal.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="configuration_groupBox">
      <property name="title">
       <string>SubDrive Configuration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>SubDrive Serial Number:</string>
           </property>
           <property name="buddy">
            <cstring>router_ssid_lineEdit_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="subdrive_sn_lineEdit_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Internet Router SSID:</string>
           </property>
           <property name="buddy">
            <cstring>router_ssid_lineEdit_2</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="router_ssid_lineEdit_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="list_pushButton_4">
           <property name="text">
            <string>List...</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/1414629197_list-unordered-128.png</normaloff>:/images/1414629197_list-unordered-128.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Internet Router Password:</string>
           </property>
           <property name="buddy">
            <cstring>router_pwd_lineEdit_3</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="router_pwd_lineEdit_3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>SubDrive Web System ID:</string>
           </property>
           <property name="buddy">
            <cstring>subdrive_web_id_lineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="subdrive_web_id_lineEdit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="advanced_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Server Communication Interval (seconds):</string>
              </property>
              <property name="buddy">
               <cstring>server_comm_interval_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="server_comm_interval_spinBox">
              <property name="maximum">
               <number>3600</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Server Address:</string>
              </property>
              <property name="buddy">
               <cstring>server_address_lineEdit_4</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="server_address_lineEdit_4">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="dhcp_radioButton">
            <property name="text">
             <string>DHCP Client</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fixed_ip__radioButton_2">
            <property name="text">
             <string>Fixed IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="fixed_ip_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>IP:</string>
                 </property>
                 <property name="buddy">
                  <cstring>server_address_lineEdit_4</cstring>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLineEdit" name="fixed_ip_IP_lineEdit_5">
                 <property name="inputMask">
                  <string>999.999.999.999</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Network Mask:</string>
                 </property>
                 <property name="buddy">
                  <cstring>server_address_lineEdit_4</cstring>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLineEdit" name="fixed_ip_net_mask_lineEdit_6">
                 <property name="inputMask">
                  <string>999.999.999.999</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Gateway:</string>
                 </property>
                 <property name="buddy">
                  <cstring>server_address_lineEdit_4</cstring>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLineEdit" name="fixed_ip_gateway_lineEdit_7">
                 <property name="inputMask">
                  <string>999.999.999.999</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>DNS Server:</string>
                 </property>
                 <property name="buddy">
                  <cstring>server_address_lineEdit_4</cstring>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLineEdit" name="fixed_ip_dns_server_lineEdit_8">
                 <property name="inputMask">
                  <string>999.999.999.999</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>dhcp_radioButton</zorder>
         <zorder>fixed_ip__radioButton_2</zorder>
         <zorder></zorder>
         <zorder>fixed_ip_frame</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttons_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_pushButton">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/document_save.png</normaloff>:/images/document_save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="edit_pushButton_2">
         <property name="text">
          <string>Edit</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/edit-icon.png</normaloff>:/images/edit-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="retrieve_pushButton">
         <property name="text">
          <string>Retrieve Config.</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/Refresh-icon.png</normaloff>:/images/Refresh-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="advanced_pushButton">
         <property name="text">
          <string>Advanced...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAboutApp"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionNetwork_Config"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="text">
    <string>SubDrive Configuration Tool...</string>
   </property>
  </action>
  <action name="actionNetwork_Config">
   <property name="text">
    <string>Network Config.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
